### DISCUSSION TITLE
pre-newbie question regarding project folders and git

### DISCUSSION THREAD

**Discussion by heiferhigh76** [community/community#22580] [New to GitHub]
<p>I have GitHub desktop and use PyCharm. There’s a lot more I could add (and I’ll be happy to write more if needed) but I think the question below is the gist of my mental block.</p>
<p>Will my code eventually be in:</p>
<p>three places (random local folder + local repository + GitHub) </p>
<p>or </p>
<p>two places (local repository + GitHub)</p>
<p>?</p>
<p>Thanks for any help.</p>

**Comment by Wabri** [comment, marked as answer]
<p>Two places: local repository and remote repository on github.</p>

**Comment by heiferhigh76** [comment]
<p>thank you so much </p>
